.github/workflows/rust-integration.yaml

```diff
@@ -0,0 +1,42 @@
+name: Continuous Integration workflow for Rust apps
+
+on:
+  workflow_call:
+
+jobs:
+  analysis:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # Rust relies on a Low-Level Virtual Machine (LLVM) to capture coverage, 
+      # and some massaging of the output binary is needed to get an LCOV file. 
+      # These tools do that processing for us.
+      - name: Install coverage dependencies
+        run: cargo install grcov && rustup component add llvm-tools
```

Comment:
`cargo install *anything*` downloads and builds locally. So this will be adding a new project to build, right?